### PR TITLE
Bookstore-styling

### PR DESCRIPTION
**Pull request by hiwina**
- Added styles to mach the Zeplin's design.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,15 +1,291 @@
...
+.right {
+  padding-right: 35px;
+}
+
+.book {
+  display: grid;
+  grid-template-columns: 2fr 1fr 1fr;
+
+  /* display: flex;
+  justify-content: space-between; */
```
Comment (divinecharlotte) [inline src/App.css]: - [ ] kindly consider removing the commented lines of codes to make your code base look professional

File: src/components/BookItem.js [hunk 2]
```diff
@@ -9,19 +12,56 @@ const BookItem = (prop) => {
...
+            <button type="button" onClick={handleDelete} className="btn-single">
+              Remove
+            </button>
 
-        </button>
-      </li>
-    </div>
+            <button type="button" onClick={() => { }} className="btn-single">
+              Edit
+            </button>
```
Comment (divinecharlotte) [inline src/components/BookItem.js]: - [ ] kindly consider sticking to the given design as much as possible to make your project an exact duplicate design of the given one check on the image below for the highlighted point to fix 😀 
Screen Shot 2023-06-24 at 21 41 55

**Review by divinecharlotte — APPROVED** [review]
Hi @hiwina  , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Branch named well👍 
- Descriptive readme 👍 

 




 

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @hiwina , 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**